Assets/_BForBoss/_Weapons/Scripts/Weapons/WeaponsManager.cs

```diff
@@ -11,6 +11,9 @@ namespace Perigon.Weapons
     public class WeaponsManager : MonoBehaviour
     {
         private const float ACCUMULATED_RECOIL_PERCENTAGE = 0.99F;
+        private const string MELEE_ANIMATOR_PARAM = "Melee";
+        private static int MeleeAnimatorID = Animator.StringToHash(MELEE_ANIMATOR_PARAM);
+        
```

Comment:
Up to you, as it is pretty unimportant now, but since we will be having enemies and bosses with their own animation, it might be better to have `MELEE_ANIMATOR_PARAM` string be something like `Melee_Player` or some thing like that!